### ISSUE TITLE
compatibility with MCJoypad mod limited and to seated mode

### ISSUE THREAD

**Issue by Nep-Soundfont** [jrbudda/Vivecraft_112#119]
well, i figured it out. theres a mod called mcjoypad somewhere on the internet that adds controller support to minecraft, as for vivecraft: worked like it was supposed to but with the two flaws of i had to use the mouse to move the HUD, which can be fixed with blank keyboard keybinds for moving the HUD and it only works in seated mode, which, honestly wasnt that much of a deal for me but might be for others, and who uses an XBox controller while standing while wearing an HMD?

in short: compatibility is there, but limited by unable to move the HUD and it's only available in seated mode

**Comment by Nep-Soundfont** [comment]
although, this is more of a compatibility report than an issue. oh well, i'll just close this now
